3scale/APIcast
# Replace luafilesystem-ffi to fix build issue on aarch64

This is a ~~experimental PR~~, replace the lfs-ffi https://github.com/3scale/luafilesystem-ffi/blob/master/lfs_ffi.lua#L921  with https://github.com/lunarmodules/luafilesystem to ~~hopefully~~ fix the  issue on arm64 https://issues.redhat.com/browse/THREESCALE-10662.

### Verification Steps

The lfs module is mainly used during startup, so the criteria is to have the APIcast start without errors.

* Build docker image from this git branch
```
make runtime-image IMAGE_NAME=apicast-test
```
* HTTP dev environment setup
```
cd dev-environments/plain-http-upstream
make gateway IMAGE_NAME=apicast-test
```

The gateway should be running without error

*  The request should be accepted (200 OK)
```
curl --resolve get.example.com:8080:127.0.0.1 -v "http://get.example.com:8080/?user_key=123"
```
